tools/build-openssl.sh

```diff
@@ -6,7 +6,7 @@
 
 # NOTE: Keep this updated to make sure we always build the latest
 #       version of OpenSSL in the 1.0 release train.
-OPENSSL_VERSION=1.0.2t
+OPENSSL_VERSION_SUFFIX=u
```

Comment:
Why? the suffix is not unique, only in combination with the version. revert this.